models/prices/bitcoin/prices_bitcoin_tokens.sql

```diff
@@ -0,0 +1,30 @@
+{{ config(
+        schema='prices_bitcoin',
+        alias = 'tokens',
+        materialized='table',
+        file_format = 'delta',
+        tags = ['static']
+        )
+}}
+SELECT 
+    token_id
+    , blockchain
+    , symbol
+    , contract_address
+    , CAST(decimals as int) as decimals
```

Comment:
if `decimals` is always null on bitcoin, you can just do `CAST(null as int) as decimals` here and avoid writing it out each time in list